redbox-core/redbox/chains/components.py

```diff
@@ -6,17 +7,42 @@
 from langchain_core.utils import convert_to_secret_str
 import tiktoken
 
+from redbox.models.chain import AISettings
 from redbox.models.settings import Settings
 from redbox.retriever import AllElasticsearchRetriever, ParameterisedElasticsearchRetriever
 
 
-def get_chat_llm(env: Settings):
-    return AzureChatOpenAI(
-        api_key=convert_to_secret_str(env.azure_openai_api_key),
-        azure_endpoint=env.azure_openai_endpoint,
-        model=env.azure_openai_model,
-        api_version=env.azure_api_version_embeddings,
-    )
+def get_chat_llm(env: Settings, ai_settings: AISettings):
```

Comment:
this is the key change

Reply:
Very clean, but should we document somewhere that the the default model will be GPT 3.5 turbo if that is that key is present in the .env over any other defined models? Just conscious that busy envs in future might cause headaches